cvat-ui/src/components/annotation-page/standard-workspace/controls-side-bar/tools-control.tsx

```diff
@@ -958,6 +963,29 @@ export class ToolsControlComponent extends React.PureComponent<Props, State> {
         );
     }
 
+    private renderObjectTypeBlock(): JSX.Element {
+        const { selectedObjectType } = this.state;
+        const objectTypes = Object.values(ObjectType);
+        objectTypes.splice(objectTypes.indexOf(ObjectType.TAG), 1);
```

Comment:
If you mean that this code supports only `shape` and `track`, you should write exactly this.
Otherwise, when we will add one more object type, this part will be automatically broken and nobody will know about it